calico-cloud_versioned_docs/version-18/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,51 +2,52 @@
 description: Understand scan results in Manager UI.
 ---
 
-# Understand scan results in Manager UI
+# View scanned and running images
 
-In the left navbar in Manager UI, click **Image Assurance**, **Scan Results**.
+The Images Assurance board in Manager UI provides separate lists for *scanned images* and *running images*. Separating scanned images from runtime images provides the most flexibility, including exporting the running images into your own scanner solution.
 
-**Scan Results tab**
+In the left navbar in Manager UI, click **Image Assurance**, **All Scanned Images**.
 
-The left panel lists scanned images (assuming you set up the CLI scanner to report scan results to {{prodname}}).
+## All Scanned Images tab
 
-The right pane lists:
+This tab lists scanned images if you have enabled one of the [Image Assurance scanners](./scanners/overview).
 
-- Number of images grouped by Fail, Warn, Pass, and Unknown (the CLI scanner has not yet reported results)
-- High Risk Repos (top five repos based on total Fails)
-- Image details (when you have an image selected from the left panel list) lists associated CVEs, if any
-    - For each CVE listed, you can view the details of the vulnerability or add an exception to ignore the vulnerability (see [Create vulnerability exceptions](#create-vulnerability-exceptions)). 
+## Running Images tab
 
-**Runtime tab**
+The tab lists active container images *for all connected managed clusters*. It provides the CVEs associated with running pods to help you assess pod vulnerability. If you are using your own scanner, you can export running image URLs to your backend system. This tab is disabled by default.
```

Comment:
I was initially confused with the wording here, but I then assumed it's referring to the export section below. Should we link the section there in this sentence?

Reply:
Removed this sentence: If you are using your own scanner, you can export running image URLs to your backend system.